html/Corporate/style.scss

```diff
@@ -0,0 +1,664 @@
+@import url("https://fonts.googleapis.com/css?family=Exo:300,400,700&display=swap");
+
+$font-primary: "Exo", sans-serif;
+$color-primary: #3a3a3a;
+$color-secondary: #f04c4c;
+$section-bg-color: #fcfcfb;
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: $font-primary;
+  box-sizing: border-box;
+}
+
+@media screen and (min-width: 1024px) {
+  html {
+    font-size: 1.2rem;
+  }
+}
+
+body {
+  color: $color-primary;
+}
+
+h2 {
+  font-weight: lighter;
+}
+
+@mixin section-type() {
+  color: $color-secondary;
+  font-size: 0.6rem;
+  text-transform: uppercase;
+  letter-spacing: 3.2px;
+}
+
+@mixin section-title() {
+  font-size: 1.7rem;
+  letter-spacing: 1.6px;
+  font-weight: normal;
+}
+
+@mixin section-subtitle() {
+  font-size: 1.2rem;
+  font-weight: normal;
+}
+
+@mixin section-description {
+  font-size: 0.8rem;
+  line-height: 1.6;
+  opacity: 0.8;
+}
+
+@mixin vertical-padding($top: 100px, $bottom: 100px) {
+  padding-top: $top;
+  padding-bottom: $bottom;
+}
+
+@mixin default-padding ($left: 10%, $right: 10%, $top: 100px, $bottom: 100px) {
+  padding-left: $left;
+  padding-right: $right;
+  padding-top: $top;
+  padding-bottom: $bottom;
+}
+
+// -------------- Banner --------------
+.banner {
+  @include default-padding();
+  @include vertical-padding(200px, 200px);
+  background: url(./assets/banner-img/photo-1433354359170-23-a-4-ae-7338-c-6@3x.jpg)
+    right bottom/cover no-repeat;
+  text-align: center;
+
+  &__container {
+    margin: 0 auto;
+  }
+
+  &__title {
+    font-size: 2rem;
+  }
+
+  &__play {
+    margin: 20px 0;
+    background: transparent;
+    border: none;
+
+    img {
+      width: 40px;
+      opacity: 0.8;
+      cursor: pointer;
+    }
+  }
+
+  &__try-now {
+    display: block;
+    margin: 0 auto;
+    padding: 10px 35px;
+    color: #ffffff;
+    background: $color-secondary;
+    border: none;
+    border-radius: 3px;
+    letter-spacing: 3px;
+    text-transform: uppercase;
+    cursor: pointer;
+  }
+
+  &__description {
+    margin-top: 10px;
+    font-size: 0.7rem;
+  }
+}
+
+@media screen and (min-width: 1024px) {
```

Comment:
it is posible to write media queries right under class properties, same applies below
```
.banner {
  // your css
  @media screen and (min-width: 1024px) {
    // your media changes
  }
}
```